java/ql/test/stubs/jackson-databind-2.10/com/fasterxml/jackson/annotation/JsonTypeInfo.java

```diff
@@ -0,0 +1,27 @@
+package com.fasterxml.jackson.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.ANNOTATION_TYPE, ElementType.TYPE, ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface JsonTypeInfo {
+    JsonTypeInfo.Id use();
+
+    public static enum Id {
+        CLASS("@class"),
+        MINIMAL_CLASS("@c");
+
+        private final String _defaultPropertyName;
+
+        private Id(String defProp) {
+            this._defaultPropertyName = defProp;
+        }
+
+        public String getDefaultPropertyName() {
+            return this._defaultPropertyName;
+        }
```

Comment:
This does not seem like a minimal stub. We shouldn't accidentally rely on implementation in our tests, so we better simplify:
```suggestion
        private Id(String defProp) { }

        public String getDefaultPropertyName() { return null; }
```

Reply:
@aschackmull Sure, I've applied the suggestion. Thanks!